### PR TITLE
Redact the things

### PR DESCRIPTION
**Pull request by jimmynotjim**
████████ ████ ████ 
 
## Additions
 
- ██████
 
## Removals
 
- ████████
 
## Changes
 
- ████ 
- ██████
 
## Testing
 
- ████ ████ ████████ 
 
## Review
 
- @███
- @██████
- @████
 
## Preview
 
████ ██████ ██
 
## Notes
 
- ████ ██ ████████

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
                                 <a class="list_link" href="/newsroom/">Newsroom</a>
                             </li>
                             <li class="list_item list-expanding_list-item">
-                                <a class="list_link" href="/contact-us/">Contact Us</a>
+                                <a class="list_link" href="#">TBD Content</a>
```
Comment (anselmbradford) [inline _includes/header.html]: Should this have `list_link__disabled` class added?

Comment (jimmynotjim) [inline _includes/header.html]: In theory, but we'd have to rewrite how `list_link__disabled` works. Right now it's relying on being a link directly related to the top level nav as `.primary-nav_top-level-list>.list_item>.list_link__disabled`. Refactoring that is outside our ability to get this up for review this morning.

Comment (anselmbradford) [inline _includes/header.html]: Perhaps just throw in some inline style to disable the hover-state? Right now it's not like the other TBAs in that it has the same style as the items around it and isn't greyed out.

**Comment by anselmbradford** [comment]
Footer also needs "TBA Content" instead of "TBA" based on what @jimmynotjim said in chat IIRC.

**Comment by jimmynotjim** [comment]
I think that's everything. Someone give me a :+1: and I'll squash this down and get it into FJ for that PR

**Comment by anselmbradford** [comment]
:+1: Incorporates https://github.com/cfpb/cfgov-refresh/pull/390

**Comment by sebworks** [comment]
lol redacted

**Pull request merged**